.github/workflows/golangci-lint.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
The linter is still failing with [this error](https://github.com/rook/rook/pull/11427#pullrequestreview-1214345681). You may want to revert the linter update for now unless you can find a fix.

Reply:
looks like issue is with linter version